src/download.tsx

```diff
@@ -143,11 +144,11 @@ class Download extends KalturaPlayer.core.BasePlugin {
     await this.ready;
 
     this.downloadPluginManager.setShowOverlay(false, false);
-    const downloadMetadata = await this.downloadPluginManager.getDownloadMetadata(true);
+    this.downloadMetadata = await this.downloadPluginManager.getDownloadMetadata(true);
```

Comment:
we should also clean up the value, probably in reset()

Reply:
ouddated